.github/workflows/unit-tests.yml

```diff
@@ -113,13 +113,19 @@ jobs:
         with:
           path: ${{ github.run_id }}
 
+      - name: Check for notebook changes
+        id: check_notebooks
+        run: check_notebooks=$(git diff --quiet main HEAD -- '*.ipynb' && echo false || echo true) >> $GITHUB_ENV
```

Comment:
this would be correct

```
check_notebooks=$(git diff --quiet main HEAD -- '*.ipynb' && echo false || echo true)
echo "check_notebooks=$check_notebooks" >> $GITHUB_ENV
```